tests/ludwig/features/test_timeseries_feature.py

```diff
@@ -0,0 +1,36 @@
+from typing import Dict
+
+import pytest
+import torch
+
+from ludwig.features.timeseries_feature import TimeseriesInputFeature
+
+SEQ_SIZE = 2
+TIMESERIES_W_SIZE = 1
+MAX_LEN = 7
+EMBEDDING_SIZE = 5
+DEVICE = "cuda" if torch.cuda.is_available() else "cpu"
+
+
+@pytest.fixture(scope="module")
+def timeseries_config():
+    return {
+        "name": "timeseries_12",
+        "type": "timeseries",
+        "max_len": MAX_LEN,
+        "embedding_size": EMBEDDING_SIZE,
+        "max_sequence_length": SEQ_SIZE,
+        "fc_size": 8,
+        "state_size": 8,
+        "num_filters": 8,
+        "hidden_size": 8,
+    }
+
+
+@pytest.mark.parametrize("encoder", ["rnn", "stacked_cnn", "parallel_cnn"])
```

Comment:
Instead of defining a list here, could we use `tests.integration_tests.utils.ENCODERS`?

Reply:
Not all encoders are needed for timeseries, correct?

Reply:
I'm not sure. Perhaps when we revamp the documentation, we should list the encoders that timeseries features use.